test/test_support.ts

```
@@ -173,6 +173,19 @@ export class GoldenFileTest {
         .map(f => path.join(this.path, GoldenFileTest.tsPathToJs(f)));
   }
 
+  /**
+   * Find the absolute path to the tsickle root directory by reading the
+   * symlink bazel puts into bazel-bin back into the test_files directory
+   * and chopping off the test_files portion.
+   */
+  get workspaceRoot(): string {
+    if (!this.tsFiles) {
```

do not silently swallow errors, fail loudly.